src/main/java/core/basesyntax/lotteryball/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.lotteryball;
+
+import java.util.Random;
+
+public class Lottery {
+    private Ball ball;
+
+    public Ball getRandomBall() {
+        ball = new Ball();
+        ball.setBallColor(new ColorSupplier().getRandomColor());
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method